httputil/request_test.go

```diff
@@ -0,0 +1,76 @@
+package httputil
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
This is a little bit pedantic, but I think `gofmt` would format these imports like:

```go
import (
        "net/http"
        "testing"

        "github.com/stretchr/testify/assert"
)
```